docs/community.rst

```diff
@@ -1,4 +1,35 @@
 .. _community:
 
 Community
-=========
\ No newline at end of file
+=========
+
+.. _code_of_conduct:
+
+Code of conduct
+---------------
+
+This project has adopted the `GitHub community guidelines 
+<https://help.github.com/en/github/site-policy/github-community-guidelines>`_.
+
+.. _communication:
+
+Real-time communication
+-----------------------
+
+For real time communication please join us on
+`Gitter <https://gitter.im/fairlearn/community>`_.
+All are welcome!
+
```

Comment:
Should we also mention GitHub issues and Stack Overflow similarly to: https://github.com/fairlearn/fairlearn#issues

Reply:
This page kind of says what goes where and I've added a link to it now https://github.com/fairlearn/fairlearn/issues/new/choose

Reply:
Works for now.